projects/core/src/user/facade/user.service.spec.ts

```diff
@@ -400,6 +404,60 @@ describe('UserService', () => {
     );
   });
 
+  describe('update personal details', () => {
+    const username = 'xxx';
+    const userDetails: User = {
+      uid: username,
+    };
+
+    it('should dispatch UpdateUserDetails action', () => {
+      service.updatePersonalDetails(username, userDetails);
+      expect(store.dispatch).toHaveBeenCalledWith(
+        new fromStore.UpdateUserDetails({ username, userDetails })
+      );
+    });
+
+    it('should return the loading flag', () => {
+      spyOn(service, 'getUpdatePersonalDetailsResultLoading').and.returnValue(
+        of(true)
+      );
```

Comment:
You can't mock the function being tested.

Reply:
oops